backend/src/plugins/AFK/commands/AFKCmd.ts

```diff
@@ -0,0 +1,46 @@
+import { afkCmd } from "../types";
+import { commandTypeHelpers as ct } from "../../../commandTypes";
+import { sendErrorMessage, sendSuccessMessage } from "../../../pluginUtils";
+import { parseStatusMessage } from "../functions/parseStatusMessage";
+
+export const AfkSetCmd = afkCmd({
+    trigger: ['afk', 'afk set'],
+    permission: 'can_afk',
+
+    signature: {
+        status: ct.string({ rest: true, required: true }),
+    },
+
+    async run({ message: msg, args, pluginData }) {
+        // Checks if the user is AFK, if so, return.
+        const isAfk = await pluginData.state.afkUsers.getUserAFKStatus(msg.author.id);
+        if (isAfk) return;
+    
+        const status = args.status.join(" ");
+
+        // Check status length
+        if (status.length > 124) {
```

Comment:
For clarification, I did this in order to prevent people from posting wall-text using their afk statuses.